slang-checks/src/main/java/com/sonarsource/slang/checks/utils/SyntacticEquivalence.java

```diff
@@ -24,6 +24,11 @@
 import com.sonarsource.slang.api.LiteralTree;
 import com.sonarsource.slang.api.NativeTree;
 import com.sonarsource.slang.api.Tree;
+import com.sonarsource.slang.visitors.TreePrinter;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.annotation.Nullable;
 
 public class SyntacticEquivalence {
 
```

Comment:
The `areEquivalent` method should also be updated accordingly with the new types of Tree added in this PR

Reply:
I think it's ok.